_includes/supporter.html

```diff
@@ -10,8 +10,8 @@
       and maintain the inventory for the conference.
     </div>
     <p class="description">
-      Reach out to us at
-      <a href="mailto:sponsor@in.pycon.org">sponsor@in.pycon.org</a>
+      Take a look at
+      <a href="{{ '/sponsorship-prospectus' | relative_url }}">Sponsorship Page</a>
```

Comment:
Can we rename the page to `/sponsor`?

Reply:
/sponsor page will be used for listing out the sponsors.